local_planner/src/nodes/histogram.cpp

```diff
@@ -19,27 +13,19 @@ void Histogram::upsample() {
   resolution = resolution / 2;
   z_dim = 2 * z_dim;
   e_dim = 2 * e_dim;
-  std::vector<std::vector<double> > temp_bin;
-  std::vector<std::vector<double> > temp_age;
-  std::vector<std::vector<double> > temp_dist;
-  temp_bin.resize(e_dim);
-  temp_age.resize(e_dim);
-  temp_dist.resize(e_dim);
-  for (int i = 0; i < e_dim; ++i) {
-    temp_bin[i].resize(z_dim);
-    temp_age[i].resize(z_dim);
-    temp_dist[i].resize(z_dim);
-  }
+  Eigen::MatrixXd temp_age;
```

Comment:
Can we use floats instead? I don't think we need the precision, and depending on the hardware there might be a significant improvement in performance

Reply:
Also, I'm not familiar with Eigen, but it seems like there should be a way to declare a Matrix with the size right away, avoiding a call to the `resize` method?